src/main/java/step1/Main.java

```diff
@@ -0,0 +1,9 @@
+package step1;
+
+public class Main {
+	public static void main(String[] args) {
```

Comment:
들여 쓰기에는 탭 문자가 사용되지 않습니다. 아래의 Google Java Style Guide를 참고해 주세요.

> Tab characters are not used for indentation.

```suggestion
    public static void main(String[] args) {
```